### DISCUSSION TITLE
Welcome to libsamplerate Discussions!

### DISCUSSION THREAD

**Discussion by evpobr** [libsndfile/libsamplerate#133] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shashwat-kumar-rmi** [comment]
Hi,
Are there any plans for apple silicon support in the future?

**Reply by evpobr** [reply]
Hi @shashwat-kumar-rmi . Сan you explain in more detail what exactly is not working?

**Reply by shashwat-kumar-rmi** [reply]
I am getting the following error while trying to compile on Mac with an M1 chip.

`checking build system type...Cfg/config.guess: unable to guess system type.`

Please check the following screen capture.

![image](https://user-images.githubusercontent.com/113004609/195301349-ce22969a-6a50-4be4-b267-46f4f20c2e2f.png)
